CandidateReranking.md

```diff
@@ -103,12 +101,13 @@ The profile is built with data from either **MY_SUPPLY** or **EDGE** (identified
 #### Profile
 | Param    | Type | Description
 |----------|------|--------|
-| document_id | String | **Required**: Unique CB identifier string for the candidate.
-| recent_jobs | [RecentJobs[]](#recentjob) | **Required**: Array of **RecentJob** objects. Recent s jobs held by the candidate. Reranker requires at least one RecentJob object.
-| skills | [Skill[]](#skill) | **Required**: Array of **Skill** objects. Skills associated with the candidate.
-| years_of_experience | Double | **Required**: Total years of experience the candidate has as a double.
+| document_id | String | **Optional**: Unique CB identifier string for the candidate.
+| recent_jobs | [RecentJobs[]](#recentjob) | **Optional**: Array of **RecentJob** objects. Recent s jobs held by the candidate. Reranker requires at least one RecentJob object.
+| skills | [Skill[]](#skill) | **Optional**: Array of **Skill** objects. Skills associated with the candidate.
+| years_of_experience | Double | **Optional**: Total years of experience the candidate has as a double.
```

Comment:
Confusing/ambiguous phrasing here (Mike, how many years of experience do you have as a double?)

Reply:
`/img stunt double mike`